### DISCUSSION TITLE
关于噼里啪啦

### DISCUSSION THREAD

**Discussion by demolute** [share121/inter-knot#3248] [其他]
![微信图片_20240820103317](https://github.com/user-attachments/assets/2dd4ff58-8cf9-40f3-8e3e-2e3d555160fa)
![微信图片_20240820103327](https://github.com/user-attachments/assets/f3cfe4e2-3626-4cf5-952b-bb322731717e)
这次新深渊我明明已经打算问候策划组全家了，小陈你tm硬是让我对原神的千言万语汇成一句：b站，我阐释你的梦！

**Comment by wjl012** [comment]
这个视频说了米哈游和b站的爱恨情仇
https://www.bilibili.com/video/BV1de4y1c7WH/?share_source=copy_web&vd_source=b428b56d4c04aa2bd2af93ee27cd4dcd

**Comment by HFOxin** [comment]
笑点解析：b站排行榜第一名《播放量276.8万》

**Comment by peppermint2005** [comment]
这次深渊不难吧，一次过的

**Comment by lllliufeng** [comment]
我还是那句话，这网站**想让你有播放量就让你有播放量**，**想压你播放量就压你播放量**。我相信噼里啪啦清汤大老爷**永远不会放过任何一个好用户**，也**永远不会冤枉任何一个坏用户**。噼里啪啦实在太好玩了，**推送机制公平**，**审核完善**，**用户素质高**，噼里啪啦就是世界上最好用的网站（

**Comment by Arocohol** [comment]
新深渊感觉还好吧，主要是加奖励了我就举双手支持
